combo/combo.py

```diff
@@ -0,0 +1,52 @@
+"""
+ID: whuan2001
+LANG: PYTHON2
+TASK: combo
+"""
+def solution():
+    fin = open("combo.in", "r")
+    fout = open("combo.out", "w")
+    N = int(fin.readline().strip())
+    a1, a2, a3 = map(int, fin.readline().strip().split())
+    b1, b2, b3 = map(int, fin.readline().strip().split())
+    if N >= 5:
+        farmer_dail1 = dail(a1, N, {})
```

Comment:
You do not need pass the empty dict from function parameter. You can just create on in the dial function and return it.
Also, you can use map function over a list to reduce the boilerplate of your code.

Reply:
I have deleted the empty dict and created dic in dail funtion.
How to use map function over a list?